### ISSUE TITLE
GRASS r.series produces wrong results when using rasters with float values

### ISSUE THREAD

**Issue by miron-t** [qgis/QGIS#31844]
When GRASS r.series is used with multiple rasters with float values, r.series does not work as expected. I tried “count” and “average” and the results are wrong (count gives wrong number, average gives a raster with 0 values). I tried the same set of rasters with QGIS 2.18.28 and the results are as expected.

QGIS-Version: 3.8.1-Zanzibar
Windows 10 (10.0)

**Comment by gioman** [comment]
@miron-t is the generated GRASS command correct?

**Comment by miron-t** [comment]
Thanks @gioman for the Idea, I checked the GRASS command in the QGIS history after running the process and I found out that the “range” parameter is the problem: even though it is optional, putting no values leads to a 'range':[0,0], so apparently all values outside this range are ignored? I tried running the same process this time with an input for range (0-10000) and it works as expected. Tried count as well as average.
